### ISSUE TITLE
Allow for track-specific (generic) content

### ISSUE THREAD

**Issue by ErikSchierboom** [exercism/mentors#24]
Currently, we are creating mentoring notes for individual exercises. This is great, but a lot of mentoring notes/tips/pointers are very generic, and apply to all exercises. A good example of this are coding style guidelines. I think it would be nice if there could be a `README.md` for each track listing such global notes.

**Comment by skade** [comment]
I agree with this. A good piece of info to add to these notes is which exercises are generally easy to mentor and can be quickly burned through in a bulk.

**Comment by rpottsoh** [comment]
[This](https://github.com/exercism/delphi/blob/master/exercises/hello-world/GETTING_STARTED_GUIDE.md) is what I am currently providing to all Delphi students when they download `hello-world`.   It is pretty detailed and intended to be reference material.

**Comment by Cohen-Carlisle** [comment]
It would be nice to have a place to put tooling as well, but in this case the `README.md` does not seem the right place.

> Cohen Carlisle [3:44 PM]
I made another little script that is designed to download an elixir exercise, enable the pending tests, and run the test suite. https://gist.github.com/Cohen-Carlisle/8b708d68b3fc11e62e3f41244791d900
Can be paired with this to check the code formatting. https://gist.github.com/Cohen-Carlisle/915d4aa8190ba222d1bbe021ce48982c

> ihid [3:46 PM]
I wonder if we should add a section of the mentoring repo for tools like this. Would be super useful to have this in a permanent place.

**Comment by budmc29** [comment]
Sounds like a great idea.

Something that's easy accessible from your mentoring exercise page, from there you can just copy it and tweak it for the user, and add for example relevant code.

I do find myself having to share the same points for multiple users a lot.

**Comment by F3PiX** [comment]
Maybe we should ask TextExpander for some kind of sponsoring, so that we can have teams per track with shared snippets.
